File: eCinema-Seminarski/eCinema/eCinema.Core/Entities/Photo.cs
namespace eCinema.Core
{
    public class Photo : BaseEntity
    {
        public Photo()
        {
            GuidId = Guid.NewGuid();
        }
        public Guid GuidId { get; set; }
        public byte[] Data { get; set; } = null!;
        public string ContentType { get; set; } = null!;
        public byte[] ThumbnailContent { get; set; } = null!;

        public ICollection<User> Users { get; set; } = null!;
        public ICollection<Movie> Movies { get; set; } = null!;
        public ICollection<Employee> Employees { get; set; } = null!;
    }
}

File: eCinema-Seminarski/UI/ecinema_admin/lib/models/photo.dart
class Photo {
  late String? contentType;
  late String? guidId;

  Photo({this.contentType, this.guidId});

  Photo.fromJson(Map<String, dynamic> json) {
    contentType = json['contentType'];
    guidId = json['guidId'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['contentType'] = contentType;
    return data;
  }
}
